R/latex.R

```diff
@@ -63,6 +63,7 @@ pdf_book = function(
     writeUTF8(x, f)
     latexmk(f, config$pandoc$latex_engine)
     unlink(with_ext(output, 'bbl'))  # not sure why latexmk left a .bbl there
+    unlink(with_ext(output, 'run.xml')) # clean up after biber
```

Comment:
Could you merge this to the previous line? The second argument of `with_ext()` can take a vector.

Reply:
Sure, done.